### PR TITLE
Modify the sign-in magic link (success) page

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5574

image

### CODE REVIEW

File: pages/signinLinkSent.js [hunk 1]
```diff
@@ -37,24 +31,22 @@ class SignInLinkSent extends Component {
...
     return (
-      <Page title="Login Link Sent" noRobots>
-        <Container pt={[4, 5]} pb={6} px={3} background="linear-gradient(180deg, #EBF4FF, #FFFFFF)" textAlign="center">
-          <Flex justifyContent="center" mb={4}>
-            <Icon size="60" />
-          </Flex>
-          <H3 as="h1" fontWeight="bold">
+      <Page title="Login Link Sent" noRobots showFooter={false}>
+        <Container pt={[4, 5]} pb={6} px={3} textAlign="center">
+          <Image src="/static/images/sign-in-illustration.png" width="624px" height="372px" />
```
Comment (Betree) [inline pages/signinLinkSent.js:36]: Did you download this image with a x2 resolution? I'm not sure if it's retina-ready

**Comment by Betree** [comment]
@SudharakaP E2E tests need to be updated

File: pages/signinLinkSent.js (hunk 1 above)
Comment (SudharakaP) [reply pages/signinLinkSent.js:36]: Ops, you are right. 🤯 I've updated the resolution. Should be good now.

**Comment by SudharakaP** [comment]
> @SudharakaP E2E tests need to be updated

I've missed updating one e2e test since we are changing the sentence `We've sent it to ${email}.` to `We've sent it to ${email}`. Thanks, it's updated now.

File: pages/signinLinkSent.js [hunk 2]
```diff
@@ -37,24 +31,22 @@ class SignInLinkSent extends Component {
     const { email } = this.props;
     const isValidEmail = email && isEmail(email);
     return (
-      <Page title="Login Link Sent" noRobots>
-        <Container pt={[4, 5]} pb={6} px={3} background="linear-gradient(180deg, #EBF4FF, #FFFFFF)" textAlign="center">
-          <Flex justifyContent="center" mb={4}>
-            <Icon size="60" />
-          </Flex>
-          <H3 as="h1" fontWeight="bold">
+      <Page title="Login Link Sent" noRobots showFooter={false}>
```
[the remark this answers is not in the document]
Comment (Betree) [reply pages/signinLinkSent.js:34]: I assume you mean the sign-in flow? Sometimes the footer is not in Figma but is still expected in the interface. Let's make 100% sure that we get explicit confirmation from design. Any link to Slack or Github conversations is welcome.

**Review by Betree — APPROVED** [review]
Looking great, all good for me to merge if we're 100% sure about https://github.com/opencollective/opencollective-frontend/pull/7803#discussion_r877804951

**Comment by SudharakaP** [comment]
> Looking great, all good for me to merge if we're 100% sure about [#7803 (comment)](https://github.com/opencollective/opencollective-frontend/pull/7803#discussion_r877804951)

This is good as per, https://opencollective.slack.com/archives/C03GU5J8YEL/p1653942837028969

**Pull request merged**